admin-api/authorization_server/internal_authorization.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Internal authorization server
+menuTitle: Internal authorization server
+weight: 1
+---
+
+# Internal authorization server
+
+PrestaShop can be used as an authorization server, as such it can handle the permissions and authentication to access the resource server.
+
+When PrestaShop is used as the authorization server it doesn't need to perform an external call to validate the access token since it already has all the required data to do so in its database, but it still performs the same role.
```

Comment:
```suggestion
When PrestaShop is used as the authorization server, it doesn't need to perform an external call to validate the access token since it already has all the required data to do so in its database, but it still performs the same role.
```